### DISCUSSION TITLE
可否加个视频的异地同步播放的功能，这个功能会小众吗？

### DISCUSSION THREAD

**Discussion by liangledecha** [AlistGo/alist#6635] [Ideas]
找了一圈都没有类似的功能，与女朋友不在一起时也想同步来看，不用语音对齐播放进度，如果加这个功能的话，以alist支持多端使用的基础应该有点点点点前景？

**Comment by Debcharon** [comment]
小衆，因為無女友

**Comment by pongfcnkl** [comment]
谷歌一下不是一大把这种软件
没啥意义

**Comment by liangledecha** [comment]
找了个不太优雅的解决方案，添加VideoTogether到alist网站中去，
Alist管理-设置-全局-自定义头部，将下面的代码粘贴到里面开启VideoTogether功能
(什么时候alist可以原生支持这个功能就好了555...)

<script src="https://2gether.video/release/extension.website.user.js"></script>
<script>
document.addEventListener("DOMContentLoaded", () => {
  const seAList = document.createElement("style");

  seAList.innerHTML = `
    .left-toolbar-box {
        left: var(--hope-space-5);
        right: auto;
    }
`;

  document.body.appendChild(seAList);
});
</script>

**Comment by anwen-anyi** [comment]
你可能需要这个？
- **https://github.com/synctv-org/synctv**

**Comment by user041018** [comment]
syncTV
